tests/acl/test_acl_l3v4v6.py

```diff
@@ -0,0 +1,1224 @@
+
+    Subclasses must provide `setup_rules` method to prepare ACL rules for traffic testing.
+
+    They can optionally override `teardown_rules`, which will otherwise remove the rules by
+    applying an empty configuration file.
+    """
+
+    ACL_COUNTERS_UPDATE_INTERVAL_SECS = 10
+
+    @abstractmethod
+    def setup_rules(self, dut, acl_table, ip_version):
+        """Setup ACL rules for testing.
+
+        Args:
+            dut: The DUT having ACLs applied.
+            acl_table: Configuration info for the ACL table.
+
+        """
+        pass
+
+    def post_setup_hook(self, dut, localhost, populate_vlan_arp_entries, tbinfo, conn_graph_facts):   # noqa F811
+        """Perform actions after rules have been applied.
+
+        Args:
+            dut: The DUT having ACLs applied.
+            localhost: The host from which tests are run.
+            populate_vlan_arp_entries: A function to populate ARP/FDB tables for VLAN interfaces.
+
+        """
+        pass
+
+    def teardown_rules(self, dut):
+        """Tear down ACL rules once the tests have completed.
+
+        Args:
+            dut: The DUT having ACLs applied.
+
+        """
+        logger.info("Finished with tests, removing all ACL rules...")
+
+        # Copy empty rules configuration
+        dut.copy(src=os.path.join(FILES_DIR, ACL_REMOVE_RULES_FILE), dest=DUT_TMP_DIR)
+        remove_rules_dut_path = os.path.join(DUT_TMP_DIR, ACL_REMOVE_RULES_FILE)
+
+        # Remove the rules
+        logger.info("Applying \"{}\"".format(remove_rules_dut_path))
+        dut.command("config acl update full {}".format(remove_rules_dut_path))
+
+    @pytest.fixture(scope="class", autouse=True)
+    def acl_rules(self, duthosts, localhost, setup, acl_table, populate_vlan_arp_entries, tbinfo,
+                  ip_version, conn_graph_facts):   # noqa F811
+        """Setup/teardown ACL rules for the current set of tests.
+
+        Args:
+            duthosts: All DUTs belong to the testbed.
+            rand_one_dut_hostname: hostname of a random chosen dut to run test.
+            localhost: The host from which tests are run.
+            setup: Parameters for the ACL tests.
+            acl_table: Configuration info for the ACL table.
+            populate_vlan_arp_entries: A function to populate ARP/FDB tables for VLAN interfaces.
+
+        """
+        dut_to_analyzer_map = {}
+        for duthost in duthosts:
+            if duthost.is_supervisor_node():
+                continue
+            loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix="acl_rules")
+            loganalyzer.load_common_config()
+            dut_to_analyzer_map[duthost] = loganalyzer
+
+            try:
+                loganalyzer.expect_regex = [LOG_EXPECT_ACL_RULE_CREATE_RE]
+                # Ignore any other errors to reduce noise
+                loganalyzer.ignore_regex = [r".*"]
+                with loganalyzer:
+                    self.setup_rules(duthost, acl_table, ip_version)
```

Comment:
Same here. In new version (202305 and above), we can check the status of ACL rule to determine if the rule is created successfully.